### PR TITLE
New VM v5.3.2 Release

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by holgerd77** [comment]
Very little time today, so I would just publish on npm once approved and do all the rest tomorrow morning.

**Review by alcuadrado — APPROVED** [review]

File: packages/vm/CHANGELOG.md
Comment (jochem-brouwer) [inline packages/vm/CHANGELOG.md:11]:
```suggestion
This is a hot-fix performance release, removing the `debug` functionality from PR [#1080](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1080) and follow-up PRs. While highly useful for debugging, this feature side-introduced a siginficant reduction in VM performance which went along unnoticed. For now we will remove since upstream dependencies are awaiting a new release before the `Belin` HF happening. We will try to re-introduce in a performance friendly manner in some subsequent release (we cannot promise on that though). The `step` event output is now also changed, with some elements not available anymore (they can be calculated on demand, though).
```

Comment (jochem-brouwer) [reply packages/vm/CHANGELOG.md:11]: Step event also changed.

Comment (alcuadrado) [reply packages/vm/CHANGELOG.md:11]: Which ones? That's not intended.

**Comment by holgerd77** [comment]
@jochem-brouwer sorry, don't have time to wait for CI another round, so will merge as-is.

Have published on npm.

File: packages/vm/CHANGELOG.md
Comment (alcuadrado) [reply packages/vm/CHANGELOG.md:11]: Just checked this again, and this is not the case. I didn't modify the code that creates `eventObj`, the object is not modified later, and is the only thing passed to the event listeners.

**Pull request merged**
